vite.config.ts

```diff
@@ -1,12 +1,20 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 import { defineConfig } from 'vitest/config';
+import os from 'node:os';
 
 export default defineConfig({
   test: {
     environment: 'node',
     globalSetup: './test/utils/global-setup.ts',
     testTimeout: 15000,
+    poolOptions: {
+      threads: {
+        minThreads: 1,
+        // leave half of CPU capacity for Chrome browser processes
+        maxThreads: Math.floor(os.cpus().length / 2),
```

Comment:
Is it possible for `os.cpus().length` to be 1? In that case `maxThreads` would be 0.

Reply:
Definitely possible. Let's wrap with `Math.max()`